docs/how-to/create-slice.rst

```diff
@@ -0,0 +1,107 @@
+How-to create a package slice
+******************************
+
+Problematic
+-----------------
+
+Are you tired of having an image weighing several MB for a software, or an image with many security vulnerabilities? In this tutorial we will show you how to avoid this with slices. In this tutorial we will create a slice name bins to only have the openssl binary and c_rehash in our image.
+
+Structure of a slice
+--------------------
+
+Here the structure of a slice : 
+
+.. code-block:: yaml
+
+    <name_of_the_slice>
+        <essential>
+        <contents>
+        <mutate>
+
+| The essential part is optional and permits you to install the dependencies of the packages you want to slice.
+| The contents part is required and permits you to keep the files from the package you want to keep in the slice.
+| The mutate part is optional and allows the reproduction of maintainer scripts. (use this https://github.com/google/starlark-go)
+
+Find dependencies of our packages
+---------------------------------
+
+So, the first step is to find the dependencies of the openssl package. We can easily do it with this command: 
+
+.. code-block:: sh
+
+    $ apt show openssl
+
+.. code-block:: sh
+
+    package: openssl
+    Version: 3.0.2-0ubuntu1.7
+    Origin: Ubuntu
+    Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
+    Original-Maintainer: Debian OpenSSL Team <pkg-openssl-devel@alioth-lists.debian.net>
+    Bugs: https://bugs.launchpad.net/ubuntu/+filebug
+    Depends: libc6 (>= 2.34), libssl3 (>= 3.0.2-0ubuntu1.2)
+
+So we see that openssl depends on libc6 and libssl3 (these slices already exist; we will see them after)
+
+Creating our slice
+------------------
+
+Now we will create our openssl slice, but first, you need to clone this repository (our slice database):
+
+.. code-block:: sh
```

Comment:
can you pls remove everything from here until the end of the file? the content is already covered in the suggested commit above